_posts/2018-07-20-snowplow-mini-0.6.0-released/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: Snowplow Mini 0.6.0 released
+title-short: Snowplow Mini 0.6.0
+tags: [snowplow-mini, docker, iglu-server, elasticsearch]
+author: Oguzhan
+category: Releases
+permalink: /blog/2018/07/20/snowplow-mini-0.6.0-released/
+---
+
+We are pleased to announce the 0.6.0 release of Snowplow Mini, our accessible "Snowplow in a box" distribution.
+
+Snowplow Mini is the complete Snowplow real-time pipeline running on a single instance, available
+for easy deployment as a pre-built AMI. Use it to:
+
+1. Set up an inexpensive and easily discardable Snowplow stack for testing your tracker and schema changes
+2. Learn about Snowplow without having to set up a horizontally-scalable, highly-available production-grade pipeline
+
+This release focuses on addressing the long-standing [migration to Docker][docker-migration-issue] including updating Elasticsearch stack. Also, as of this release, Snowplow Mini will be available in 3 different sizes; `large`, `xlarge` and `xxlarge` to meet varying purposes.
+
+Read on for:
+
+<!--more-->
+
+1. [New Iglu Server](#iglu-server)
+2. [Docker migration](#docker-migration)
+3. [Google Cloud Platform support](#gcp-support)
```

Comment:
I think gcp should be first

Reply:
also I think there should be a dedicated section for es and kibana since we were years behind